ext/ddtrace.stub.php

```diff
@@ -977,4 +977,9 @@ function dd_untrace(string $functionName, string $className = null): bool {}
      * @param int $timeout Timeout in milliseconds to wait for the flush to complete
      */
     function dd_trace_synchronous_flush(int $timeout): void {}
+
+    /**
+     * @internal
+     */
+    function dd_handle_fork(): void {}
```

Comment:
Can we please add no new functions to top-level namespace, but add to `DDTrace` or `DDTrace\Internal` as appropriate?